python/data_structures/linked_list.py

```diff
@@ -1,16 +1,32 @@
+class Node:
+ 
+    
+    def __init__(self, value, next_=None):
+        self.value = value
+        self.next = next_
+        
 class LinkedList:
     """
     Put docstring here
     """
 
     def __init__(self):
-        # initialization here
-        pass
-
-    def some_method(self):
-        # method body here
-        pass
-
+        self.head = None
+        
+    def insert(self, value):
+        #creating a new node with the correct value
+       self.head = Node(value, self.head)
+       
+    def includes(self, target_value):
+        current = self.head
+        while current:
+            if current.value == target_value:
+             return True
+         
+        current = current.next
```

Comment:
line 26 here should be indented one more level, to ensure that the while loop traverses.

Currently caught in an infinite while loop.